instructure/canvas-android
# [Student][MBL-12847] Hooked in database submissions to status on assignment details

To test:
Make a file submission, notice that when you come back to assignment details the page should update for "uploading in progress". When it succeeds, you'll see a succeeded message. When it fails (file quota limit for a group upload assignment), you'll see a failure message.

Should also update status for url/text submissions.
